docs/sources/tempo/configuration/_index.md

```diff
@@ -974,6 +974,13 @@ storage:
         # Default 1
         [blocklist_poll_tolerate_consecutive_errors: <int>]
 
+        # Polling will delete the index for a tenant if no blocks are found to
+        exist.  The empty_tenant_deletion_age configuration is used to tune how
+        quickly the poller will also delete any remaining backend objects found in the
+        tenant path.
```

Comment:
I think each line needs to be commented, judging by the other examples in this file. 
```suggestion
        # Polling will delete the index for a tenant if no blocks are found to
        # exist.  The empty_tenant_deletion_age configuration is used to tune how
        # quickly the poller will also delete any remaining backend objects found in the
        # tenant path.
```

Reply:
Good catch, thanks.